integration-tests/benchmark/keeper_test.go

```diff
@@ -308,7 +308,7 @@ func SetupAutomationBenchmarkEnv(t *testing.T, keeperTestConfig types.KeeperBenc
 		TTL: time.Hour * 720, // 30 days,
 		NamespacePrefix: fmt.Sprintf(
 			"automation-%s-%s-%s",
-			strings.ToLower(strings.Join(keeperTestConfig.GetConfigurationNames(), "_")),
+			strings.ToLower(strings.Join(keeperTestConfig.GetConfigurationNames(), "")),
```

Comment:
This test does not use multiple named configs. To keep the namespace naming consistent to as it was before, joining is being done without any separators.